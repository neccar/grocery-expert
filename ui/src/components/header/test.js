import React from "react";
import renderer from "react-test-renderer";
import SearchAppBar from ".";

test("Renders header", () => {
  const component = renderer.create(<SearchAppBar />);

  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
