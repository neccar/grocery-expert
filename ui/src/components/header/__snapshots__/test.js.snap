// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders header 1`] = `
<div
  className="Header-root-1"
>
  <header
    className="MuiPaper-root-19 MuiPaper-elevation4-25 MuiAppBar-root-10 MuiAppBar-positionStatic-14 MuiAppBar-colorPrimary-17"
  >
    <div
      className="MuiToolbar-root-46 MuiToolbar-regular-48 MuiToolbar-gutters-47"
    >
      <button
        aria-label="Open drawer"
        className="MuiButtonBase-root-56 MuiIconButton-root-50 MuiIconButton-colorInherit-51 Header-menuButton-4"
        disabled={false}
        onBlur={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex="0"
        type="button"
      >
        <span
          className="MuiIconButton-label-55"
        >
          <svg
            aria-hidden="true"
            className="MuiSvgIcon-root-59"
            focusable="false"
            role="presentation"
            viewBox="0 0 24 24"
          >
            <path
              d="M0 0h24v24H0z"
              fill="none"
            />
            <path
              d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
            />
          </svg>
        </span>
        <span
          className="MuiTouchRipple-root-120"
        />
      </button>
      <h6
        className="MuiTypography-root-68 MuiTypography-h6-85 MuiTypography-colorInherit-97 MuiTypography-noWrap-94 Header-title-5"
      >
        Grocery Expert
      </h6>
      <div
        className="Header-search-6"
      >
        <div
          className="Header-searchIcon-7"
        >
          <svg
            aria-hidden="true"
            className="MuiSvgIcon-root-59"
            focusable="false"
            role="presentation"
            viewBox="0 0 24 24"
          >
            <path
              d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"
            />
            <path
              d="M0 0h24v24H0z"
              fill="none"
            />
          </svg>
        </div>
        <div
          className="MuiInputBase-root-103 Header-inputRoot-8"
          onClick={[Function]}
        >
          <input
            className="MuiInputBase-input-113 Header-inputInput-9"
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            placeholder="Searchâ€¦"
            type="text"
          />
        </div>
      </div>
      <svg
        className="Header-logo-2"
      >
        logo.svg
      </svg>
    </div>
  </header>
</div>
`;
